services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Ensure that the `MovieSession` exists before attempting to delete it. Consider handling potential exceptions that may arise if the session does not exist.